ggml.c

```diff
@@ -5884,7 +5884,17 @@ struct ggml_tensor * ggml_view_1d(
     result->src1 = NULL;
 
     if (is_node) {
-        memcpy(result->padding, &offset, sizeof(offset));
+        ggml_scratch_save(ctx);
+
+        struct ggml_tensor * b = ggml_new_tensor_1d(ctx, GGML_TYPE_I32, 2);
+
+        GGML_ASSERT(sizeof(offset) <= ggml_nbytes(b));
+
+        memcpy(b->data, &offset, sizeof(offset));
+
+        ggml_scratch_load(ctx);
+
+        result->opt[0] = b;
```

Comment:
On `master` we now store the "view" offsets even when the tensor is not a `node`:

https://github.com/ggerganov/llama.cpp/blob/fa84c4b3e80199a5683438f062009c031a06c4fa/ggml.c#L5988-L5995

This is used by the Metal inference in `ggml-metal.m` to be able to determine the offsets during forward inference. I think you simply need to remove the `if (is_node)` checks and it should be good